.github/workflows/runtime-sync.yml

```diff
@@ -0,0 +1,84 @@
+name: AspNetCore-Runtime Code Sync
+on:
+  # Test this script using on: push
+  # push
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#scheduled-events-schedule
+    # Once per day at midnight PST (8 UTC)
+    - cron: '0 8 * * *'
+
+jobs:
+  compare_repos:
+    name: Compare the shared code in the AspNetCore and Runtime repos and notify if they're out of sync.
+    runs-on: windows-latest
+    steps:
+    - name: Checkout aspnetcore
+      uses: actions/checkout@v2.0.0
+      with:
+        # Test this script using changes in a fork 
+        # repository: 'Tratcher/aspnetcore'
+        repository: 'dotnet/aspnetcore'
+        path: aspnetcore
+    - name: Checkout runtime
+      uses: actions/checkout@v2.0.0
+      with:
+        # Test this script using changes in a fork 
+        # repository: 'Tratcher/runtime'
+        repository: 'dotnet/runtime'
+        path: runtime
+    - name: Copy
+      shell: cmd
+      working-directory: .\runtime\src\libraries\Common\src\System\Net\Http\aspnetcore\
+      env:
+        ASPNETCORE_REPO: d:\a\aspnetcore\aspnetcore\aspnetcore\
+      run: |
+        dir
+        CopyToAspNetCore.cmd
+    - name: Diff
+      shell: cmd
+      working-directory: .\aspnetcore\
+      run: |
+        mkdir ..\artifacts
+        git status > ..\artifacts\status.txt
+        git diff > ..\artifacts\diff.txt
+    - uses: actions/upload-artifact@v1
+      with:
+        name: results
+        path: artifacts
+    - name: Check and Notify
+      shell: pwsh
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        # Check the code is in sync
```

Comment:
Nit: probably better to have this in a separate file and run the file.

Reply:
I kept it consolidated for development purposes. If this action works well in master then I can see about splitting it up for long term maintenance.